rasa_core/training/interactive.py

```diff
@@ -686,6 +686,15 @@ def _collect_messages(evts: List[Dict[Text, Any]]) -> List[Message]:
     for evt in evts:
         if evt.get("event") == UserUttered.type_name:
             data = evt.get("parse_data")
+            logger.info("Those are the collected entities:" + str(
+                data["entities"]))
+            for entity in data["entities"]:
+                if "extractor" in entity:
```

Comment:
in which cases is this missing from the entitiy?

Reply:
Whenever the user manually annotated the entity

Reply:
makes sense, let's not use two if's though and just go with `if entity.get("extractor") != ...`